codestyle/org.eclipse.wst.xml.core.prefs

```diff
@@ -0,0 +1,7 @@
+eclipse.preferences.version=1
```

Comment:
I'm not sure of the value of these boiler plate files. I suppose if we get contributors using Eclipse they can open PRs to add Eclipse configs... otherwise this looks like dead code.

Reply:
removed